contracts/core/InfinityExchange.sol

```diff
@@ -45,7 +46,7 @@ NFTNFT                                                 NFTNFT
 NFTNFTNFT...........................................NFTNFTNFT 
 
 */
-contract InfinityExchange is ReentrancyGuard, Ownable, Pausable {
+contract InfinityExchange is IInfinityExchange, ReentrancyGuard, Ownable, Pausable {
```

Comment:
you would also need to add override specifier to functions that this contract implements from the interface